src/sys/aio.rs

```diff
@@ -1018,22 +1019,6 @@ pub fn aio_suspend(list: &[&AioCb], timeout: Option<TimeSpec>) -> Result<()> {
     }).map(drop)
 }
 
-impl<'a> Debug for AioCb<'a> {
```

Comment:
I think `AioCb`'s manual `Debug` implementation is better than the derived one, because it won't print unused fields.  It also doesn't print the `Buffer`, which is really just to track ownership.